src/layers/ArrayBitmapLayer.js

```diff
@@ -0,0 +1,117 @@
+import { BitmapLayer } from '@deck.gl/layers';
```

Comment:
I also wouldn't mind naming our layer  `BitmapLayer` and importing this as:

```javascript
 import { BitmapLayer as BaseBitmapLayer } from '@deck.gl/layers';
```

since really we are just extending the base implementation to allow for different photometric interpretations and support RGB buffers.